### ISSUE TITLE
Traffic Monitor doesent work with Start 11

### ISSUE THREAD

**Issue by Plutonic83** [zhongyang219/TrafficMonitor#1246]
When Start 11 Runs and the Startmenü ist aktivated over the App. You cant see Traffic Monitor on the right site in the Taskbar.

Sorry for my bad English

Greetings Plutonic

**Comment by bxfy1** [comment]
Yes, when<a href=https://www.stardock.com/products/start11/>start11</a> is used to replace Windows 11 default taskbar, traffic monitor texts in taskbar displays for 1 second, then it disappears.

**Comment by Plutonic83** [comment]
Is there a possibility that both programs run at the same time?

**Comment by rolltidehero** [comment]
I wish this worked. I would pay $$$ for a working plugin that worked with Start 11.

**Comment by Plutonic83** [comment]
Good News,

Start 11 v. 1.3 and TafficMonitor v. 1.84 working the same Time together.

**Comment by Plutonic83** [comment]
It works on my Windows 11 Desktop PC with the Update 22H2.

Plutonic

**Comment by jiangxilong945** [comment]
This issue also happened on my laptop PC，with Start11 v1.43 and TrafficMonitor v1.84.1.

**Comment by rolltidehero** [comment]
Thanks for getting this working! <3
